src/Groups/matrices/forms.jl

```diff
@@ -467,3 +468,311 @@ function issingular(f::SesquilinearForm{T}) where T
    f.descr != :quadratic && throw(ArgumentError("The form is not quadratic"))
    return GAP.Globals.IsSingularForm(f.X)
 end
+
+
+
+
+########################################################################
+#
+# From group to form #TODO: there are different approaches. Which is the best?
+#
+########################################################################
+
+# TODO 1st approach: brute force calculation
+# Algorithm furnished by Thomas Breuer, Aachen University
+# extended to quadratic by Giovanni De Franceschi, TU Kaiserslautern
+# WARNING: huge linear system !!
+
```

Comment:
Is this restricted to finite fields? To me it looks like the linear algebra approach could work just fine over any field or ZZ.
For instance it seems that magma can do it.
https://magma.maths.usyd.edu.au/magma/handbook/text/704